UnitySDK/Assets/ML-Agents/Scripts/Sensor/CameraSensorComponent.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using UnityEngine;
+
+namespace MLAgents.Sensor
+{
+    class CameraSensorComponent : SensorComponent
+    {
+        public new Camera camera;
+        public string sensorName = "CameraSensor";
```

Comment:
I think we can revisit this and try to make a unique name if adding another sensor component of the same type. It's OK for now though.